.travis.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - *no_coverage
       - *optimize
       - *no_imports
-      if: type != pull_request
+      #if: type != pull_request
```

Comment:
Are we wanting to keep this or are you just using this to verify it is fixed?

Reply:
Just to verify.